impl/c-qube/src/services/csv-adapter/parser/utils/csvreader.ts

```diff
@@ -6,10 +6,11 @@ import * as csv from 'csv-parser';
 export async function readCSV(filePath: string): Promise<string[][]> {
   return new Promise((resolve, reject) => {
     const rows: string[][] = [];
+    const configPath = 'ingest/config.json';
```

Comment:
don't hardcode this path here, don't make csvreader dependent on config path, take the quoteChar as a optional param in the function with the default value set to `'`. And wireup sending quoteChar while calling this function at the top level in services like `csvadapter.service.ts` and in other files were csvreader is being used.